### DISCUSSION TITLE
Meross MSS210 vs MSS210rtl

### DISCUSSION THREAD

**Discussion by MrHashimoto** [krahabb/meross_lan#288] [Q&A]
Hi, does anybody know the difference between the Mss 210 Smart plug and the MSS210 rtl? Only the latter can build a stable connection to HA, while the other one disconnects after a few hours and randomly reconnects for a few minutes only.
Thx for help in advance!

**Comment by krahabb** [comment]
Hello @MrHashimoto,
I don't know any about the differences in the two but from my and other experiences it happens (more than it should) some devices are 'hardware bugged' since the beginning and show inconsistent behaviors especially on the connection side.
